zebra/zapi_msg.c

```diff
@@ -1047,6 +1048,92 @@ static int zsend_table_manager_connect_response(struct zserv *client,
 	return zserv_send_message(client, s);
 }
 
+int zsend_bcast_zebra_srv6_locator_add(struct srv6_locator *locator)
```

Comment:
Just some more concern about this approach: broadcasting these srv6 zapi updates to every daemon. Is that necessary? it seems that it would be better to have any daemon who _can_ use that info request it and/or register for updates to it.

Reply:
> Is that necessary?

Yes. actually this is not necessary.
But this bcast is only occurred when operator is using SRv6.

In almost SRv6 router's case, operator will uses bgpd and isisd (or ospfd), and they (bgpd,isisd) should to subscribe srv6 locator information.

Thus, "make to be able to register" looks good, but I feel this is too much work. because this bcast is only occured when SRv6 is used.

Reply:
So I am concerned about this path. Zebra is not the central configuration manager - it's got a job, and that job is to manage the rib and fib (and interact with the OS/dataplane). we have other examples of configuration that is shared among multiple daemons, and that is the pattern that needs to be followed. "I feel this is too much work" isn't really convincing- it's the way it is in FRR. I feel it's too much work to maintain every special case and exception that gets introduced - that imposes a burden on the rest of the community, and that's not really acceptable IMO.